src/SprykerEco/Zed/Heidelpay/Communication/Plugin/Checkout/Oms/Condition/IsCaptureApprovedPlugin.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Heidelpay\Communication\Plugin\Checkout\Oms\Condition;
+
+use Orm\Zed\Heidelpay\Persistence\SpyPaymentHeidelpayTransactionLog;
+use Orm\Zed\Sales\Persistence\SpySalesOrderItem;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\Oms\Dependency\Plugin\Condition\ConditionInterface;
+use SprykerEco\Shared\Heidelpay\HeidelpayConfig;
+
+/**
+ * @method \SprykerEco\Zed\Heidelpay\Business\HeidelpayFacade getFacade()
```

Comment:
Please run the cs sniffer and other tools that new support also ECO
Here it should be auto-fixed to the interface: HeidelpayFacadeInterface getFacade()
as that one contains the actual contracts (@ api descr).